.circleci/config.yml

```diff
@@ -50,6 +50,7 @@ jobs:
             - v1-test-
       - run: lein deps
       - run: lein check
+      - run: lein install
```

Comment:
Why was this change necessary?

Reply:
One of the tests failed trying to use the snapshot version of lein-monolith: https://app.circleci.com/pipelines/github/amperity/lein-monolith/279/workflows/f847cab8-9497-4aab-bca5-ec2a1a4184e6/jobs/1074

I suspect this is because I updated the plugin version in the example monolith project to use the snapshot version, and seemed reasonable to install if loading the example project tries to download the plugins